client/volume_list.go

```diff
@@ -0,0 +1,22 @@
+package client
+
+import (
+	"context"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+// VolumeList returns the list of volumes.
+func (client *APIClient) VolumeList(ctx context.Context) (*types.VolumeListResp, error) {
+	resp, err := client.get(ctx, "/volumes", nil, nil)
+	if err != nil {
```

Comment:
The original function missed the err judging and it is a severe bug.
This pr already covered this. Just mark it. 👍 🍺